config/config-logging.yaml

```diff
@@ -10,24 +10,7 @@ metadata:
     bindings.labs.vmware.com/release: devel
 
 data:
-  _example: |
-    ################################
-    #                              #
-    #    EXAMPLE CONFIGURATION     #
-    #                              #
-    ################################
-
-    # This block is not actually functional configuration,
-    # but serves to illustrate the available configuration
-    # options and document them in a way that is accessible
-    # to users that `kubectl edit` this config map.
-    #
-    # These sample configuration options may be copied out of
-    # this example block and unindented to be in the data block
-    # to actually change the configuration.
-
-    # Common configuration for all Knative codebase
-    zap-logger-config: |
+  zap-logger-config: |
```

Comment:
log output should be sent to stderr, not stdout

Reply:
Updated

Reply:
Are you saying it should be `stderr` for both?

```
"outputPaths": ["stderr"],
"errorOutputPaths": ["stderr"],
```

Reply:
I see what you mean, stdout is meant to be program output and stderr is meant to be... "everything else"